src/bin/command.ts

```diff
@@ -0,0 +1,200 @@
+#!/usr/bin/env node
+
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import chalk = require('chalk');
+import {coerceOption} from '../util/coerce-option';
+import {GitHubReleaseOptions} from '../github-release';
+import {ReleasePROptions} from '../release-pr';
+import {getReleaserNames} from '../releasers';
+import main from '../runner';
+import * as yargs from 'yargs';
+
+export interface ErrorObject {
+  body?: object;
+  status?: number;
+  message: string;
+  stack: string;
+}
+
+interface YargsOptions {
+  describe: string;
+  choices?: string[];
+  demand?: boolean;
+  type?: string;
+  default?: string | boolean;
+}
+
+interface YargsOptionsBuilder {
+  option(opt: string, options: YargsOptions): YargsOptionsBuilder;
+}
+
+export default yargs
+  .config('config')
```

Comment:
I think `'config'` is the default?
```suggestion
  .config()
```